src/models/QuestionaryEditorModel.ts

```diff
@@ -90,14 +90,18 @@ export default function QuestionaryEditorModel(
               step.topic.id.toString() ===
               action.payload.destination.droppableId
             );
-          });
+          }) as TemplateStep;
```

Comment:
Is it 100% the `draft.steps` contains the `droppableId`? Otherwise, I suggest checking `to` and `from`.

Reply:
I am working on this right now and will change the approach to use integers with updating all later items. I will go trough this again. I better put this as WIP.